.gitmodules

```diff
@@ -8,8 +8,8 @@
 	branch = section_name_env
 [submodule "intel-device-plugins-for-kubernetes"]
 	path = intel-device-plugins-for-kubernetes
-	url = https://github.com/rojkov/intel-device-plugins-for-kubernetes.git
-	branch = qat2
+	url = https://github.com/intel/intel-device-plugins-for-kubernetes.git
+	branch = master
```

Comment:
Is this change actually necessary for the previous PR #6 to work? If yes, we should separate it.

Reply:
it's not necessary since the needed commit is present in both repos, but submitted a separate PR anyway.